mat/svd.go

```diff
@@ -182,11 +182,12 @@ func (svd *SVD) Values(s []float64) []float64 {
 // min(m,n) columns are the left singular vectors and correspond to the singular
 // values as returned from SVD.Values.
 //
-// If dst is not nil, U is stored in-place into dst, and dst must have size
-// m×m if the full U was computed, size m×min(m,n) if the thin U was computed,
-// and UTo panics otherwise. If dst is nil, a new matrix of the appropriate size
-// is allocated and returned.
-func (svd *SVD) UTo(dst *Dense) *Dense {
+// If dst is empty, UTo will resize dst to be m×m if the full U was computed
+// and size m×min(m,n) if the thin U was computed. If dst is non-empty, then
+// UTo will panic if dst is not the appropriate size. UTo will also panic if
```

Comment:
"When dst is non-empty, UTo will panic if dst is not the appropriate size."

Reply:
Done